### ISSUE TITLE
[Client] Configuration of WIFI connection

### ISSUE THREAD

**Issue by indisoluble** [comodoo/meta-comodoo#16]
At this moment (commit [a438268](https://github.com/indisoluble/meta-comodoo/commit/a4382687cc934a23d65f1b17f5e6cd0c16e628ee)), our image does not provide any easy way to connect to a WIFI. We should address this problem, I was thinking on a shell script that will allow us to configure a WIFI connection in a couple of easy steps from a SSH terminal.

**Comment by indisoluble** [comment]
Not sure if the solution described above worths a try. Our goal (at least on the client side) is to provide a POS ready to be used, to be more specific, ready to be used as soon as it is switched on. A complete POS will have a display & a keyboard, so there will be no need of a SSH connection but, nevertheless, the WIFI connection will have to be configured one way or another and it will have to be done before launching the browser. I am not sure we can o we should ask a user to deal with a pure console program, what is more they should not have access at all to the console. To sum, I am not sure if it is worthy to spend time in this story until we know what we really need.

*On hold* until further discussion with @javiroman.

**Comment by indisoluble** [comment]
Done: [last commit 032f522](https://github.com/comodoo/meta-comodoo/commit/032f52250c6f2b7487f0256b71f836bcd50d7335).

However there are still room for improvements, in fact I do not think the final result has the required quality. It works but the experience is not *smooth* and there are some bugs:

> * Type a passphrase smaller than 8 characteres:
>   In this case, when the list of available WIFIs is shown again,
>   yours is shown with a red circle and there is no way to connect
>   to it, even if you type the right password.
>   If you are in this situation: 1º make sure your passphrase has
>   between 8 and 64 characters, 2º restart the device, 3º try
>   again with the right passphrase.
> 
> * WIFI is disconnected or the decive lost connection for a short
>   period of time:
>   It should reconnect on its own but it does not. I am afraid this
>   is a WIP so, until we fix this problem, you have to reboot your
>   device. On the bright side, it will automatically reconnect after
>   the reboot.

But it is good enough for the moment, at least for a first beta of the complete product.

We will continue the work with story #14.
